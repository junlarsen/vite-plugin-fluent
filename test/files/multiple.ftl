greet = Hello { $name }!
product-name = YouTube