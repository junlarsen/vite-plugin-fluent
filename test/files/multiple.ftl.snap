import type { FluentResource, FluentBundle, FluentVariable } from "@fluent/bundle";
export declare const resource: FluentResource;
export declare function formatMessage(bundle: FluentBundle, id: "greet", args: {
    readonly "name": FluentVariable;
}, error?: null | Error[]): string;
export declare function formatMessage(bundle: FluentBundle, id: "product-name", error?: null | Error[]): string;
